node-gui/src/main_window/mod.rs

```diff
@@ -307,28 +339,34 @@ impl MainWindow {
                     wallet_id,
                     action: EncryptionAction::Lock,
                 });
-                Command::none()
+                Task::none()
             }
 
             MainWindowMessage::MainWidgetMessage(MainWidgetMessage::TabsMessage(
                 TabsMessage::WalletMessage(wallet_id, WalletMessage::Unlock),
             )) => {
-                self.active_dialog = ActiveDialog::WalletUnlock { wallet_id };
-                Command::none()
+                self.active_dialog = ActiveDialog::WalletUnlock {
+                    wallet_id,
+                    state: Default::default(),
+                };
+                Task::none()
             }
 
```

Comment:
Wallet unlocking doesn't work - the input field is disabled:
![node_gui_wallet_unlock_disabled](https://github.com/user-attachments/assets/147b5547-7f84-4abc-a4bc-2d95ffc366fe)

Btw, something happened to the close button on all dialogs:
![node_gui_close_button](https://github.com/user-attachments/assets/86af1ffa-5d3e-4081-9988-ef5330469e67)